package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Download course from educative.io",
   "main": "src/app.ts",
   "dependencies": {
-    "puppeteer": "^22.6.3"
+    "@types/node-fetch": "^2.6.12",
+    "node-fetch": "^3.3.2",
```

Comment:
Why do we need `node-fetch`? We are using Bun, and it has `fetch` API implemented to it. It feels like only having the type definition is enough.

_Note:  NodeJS now has `fetch` API built into it.